PancakeSwap-Bots/Set-Masterchef-Settings/src/utils.ts

```diff
@@ -0,0 +1,76 @@
+import { utils } from "ethers";
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+
+const createFinding = (call: utils.TransactionDescription): Finding => {
+  if (call.name == "setMigrator") {
+    return Finding.fromObject({
+      name: "MasterChef Settings",
+      description: `${call.name} function called in MasterChef contract.`,
+      alertId: "CAKE-5-1",
+      severity: FindingSeverity.Info,
+      type: FindingType.Info,
+      protocol: "MasterChef",
```

Comment:
just to make this the same with other bots, let's make protocol name "PancakeSwap"